src/doc/trpl/testing.md

```diff
@@ -504,3 +504,5 @@ you add more examples.
 
 We haven’t covered all of the details with writing documentation tests. For more,
 please see the [Documentation chapter](documentation.html)
+
+One final note: Tests *cannot* be run on the binary (main) file. To see more on file arrangement see the [Crates and Modules](https://doc.rust-lang.org/stable/book/crates-and-modules.html) Section
```

Comment:
A few things:

1. Can you make the link relative, please. See the line right about your diff.
2. Section should be lower case, at the end of the sentence.
3. End with a period please!
4. I mean, you can run tests *on* a binary, you can't urn unit tests defined _in_ the binary. You can test binaries through integration tests. So I'm not sure about this wording.
5. I would prefer to just remove the `main` bit, and make it "a binary" rather than "the binary (main) file"